aspnetcore/blazor/security/webassembly/azure-active-directory-groups-and-roles.md

```diff
@@ -237,6 +403,11 @@ There's no need to provide code to remove the original `groups` claim, if presen
 Register the factory in `Program.Main` (`Program.cs`) of the standalone app or *`Client`* app of a hosted Blazor solution. Consent to the `Directory.Read.All` permission scope as an additional scope for the app:
 
 ```csharp
+using Microsoft.AspNetCore.Components.WebAssembly.Authentication;
+using Microsoft.Extensions.Configuration;
+
+...
```

Comment:
What does the "..." imply here?

Reply:
More code that's skipped. We use the convention all the time. Readers seem ok with it.